core-bundle/src/Resources/config/services.yml

```diff
@@ -938,8 +928,6 @@ services:
         autoconfigure: false
         arguments:
             - '@contao.translation.data_collector_translator.inner'
-        tags:
-            - { name: kernel.reset, method: reset }
```

Comment:
This will not work as this service has disabled auto configuration.

Reply:
Nice catch. 👍

Reply:
any reason not to enable `autoconfiguration` for that file?

Reply:
I'm wondering the same. It was introduced here but not sure why: https://github.com/contao/contao/commit/ee82ee499f7cc5e305913a41866a8dc4134496f4